### PR TITLE
doc: add mention of Managed Kafka

### PR DESCRIPTION
**Pull request by mhdawson**
Add mention of Managed Kafka as an option versus
installing a Kafka instance into the cluster.

Signed-off-by: Michael Dawson <mdawson@devrus.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,3 +86,28 @@ You can also try the [KafkaJS](https://github.com/tulios/kafkajs/) version:
...
+set up to use service bindings, and to connect your Kafka instance.
+Provided you have installed the OpenShift Application Services and
+Service Binding Operators and have configured them with your RHOSAK
+information you can simply drag the arrow from the producer and consumer
+on the OpenShift topoloy page onto icon for the Kafka instance.
+
+![dragging to connect Kafka](images/drag-to-connect.png)
+
+This is possible due to the use of the
+kube-service-bindings](https://github.com/nodeshift/kube-service-bindings)
```
Comment (mhdawson) [inline README.md:105]:
```suggestion
[kube-service-bindings](https://github.com/nodeshift/kube-service-bindings)
```

File: README.md [hunk 2]
```diff
@@ -86,3 +86,28 @@ You can also try the [KafkaJS](https://github.com/tulios/kafkajs/) version:
...
+
+Instead of installing a Kafka instance into your cluster you can also
+use a managed Kafka instance like
+[Red Hat OpenShift Streams for Apache Kafka (RHOSAK)](https://developers.redhat.com/products/red-hat-openshift-streams-for-apache-kafka/overview).
+If you are deploying into OpenShift and using RHOSAK, the example is all
+set up to use service bindings, and to connect your Kafka instance.
+Provided you have installed the OpenShift Application Services and
+Service Binding Operators and have configured them with your RHOSAK
+information you can simply drag the arrow from the producer and consumer
+on the OpenShift topoloy page onto icon for the Kafka instance.
```
Comment (helio-frota) [inline README.md:100]: nit typo topology, other than that LGTM.

**Review by helio-frota — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (mhdawson) [inline README.md:100]:
```suggestion
on the OpenShift topology page onto icon for the Kafka instance.
```

**Comment by mhdawson** [comment]
@helio-frota fixed.

**Review by helio-frota — APPROVED** [review]

**Pull request merged**